src/app/dashboard/notifications/components/Notification.tsx

```diff
@@ -35,30 +38,35 @@ const Messages = () => {
 }
 
 const ModalNotification = () => {
+ const { Notifications,  } = useNotificationStore()
+
+  // useEffect(() => {
+  //   // Fetch notifications when the component mounts
+  //   useNotificationStore.getState().fetchNotifications();
+  // }, []);
   return (
     <main className="flex flex-col gap-4   text-xs h-[400px] overflow-y-scroll px-2 ">
-      {notifications.map((item) => {
-        if (item.id < 7) {
+      {notifications.map((notification) => {
+        if (notification.id < 7) {
           return (
             <div
-              key={item.id}
-              className={`flex justify-between gap-8 items-center  py-2 rounded-3xl px-2 border border-primary_8 ${
-                !item.read ? ' ' : 'opacity-30'
-              }`}
+              key={notification.id}
+              className={`flex justify-between gap-8 items-center  py-2 rounded-3xl px-2 border border-primary_8 ${!notification.read ? ' ' : 'opacity-30'
+                }`}
             >
               <div className="flex items-center gap-2">
                 <Image
-                  src={item.avatar}
-                  alt={`${item.name} avatar`}
+                  src={notification.avatar}
+                  alt={`${notification.name} avatar`}
                   className="w-[12%]"
                 />
                 <div>
-                  <h4 className="text-[#F0F0F0]">{item.name}</h4>
-                  <span className="text-[#92929D]">{item.msg}</span>
+                  <h4 className="text-[#F0F0F0]">{notification.name}</h4>
```

Comment:
Please why do we still have hard coded color values here. All possible colors for this project have been oncluded in tailwind config file, look at it an use the right variable please